SCClassLibrary/Common/GUI/tools/ServerMeter.sc

```diff
@@ -238,9 +238,15 @@ ServerMeter {
 		numIns = numIns ?? { server.options.numInputBusChannels };
 		numOuts = numOuts ?? { server.options.numOutputBusChannels };
 
-		window = Window.new(server.name ++ " levels (dBFS)",
-			Rect(5, 305, ServerMeterView.getWidth(numIns, numOuts), ServerMeterView.height),
```

Comment:
```supercollider
Rect(5, 305, ServerMeterView.getWidth(numIns, numOuts), ServerMeterView.height).moveTo(0, Window.availableBounds.height - ServerMeterView.height),
```

Reply:
Okay, I see now that you _are_ trying to offset thee servermeter by the size of the freqscope. 
Maybe we can just change the hardcoded number for now:
```supercollider
Rect(5, 400, ServerMeterView.getWidth(numIns, numOuts), ServerMeterView.height),
```
or something.